.github/workflows/codeql-codescan.yml

```
@@ -0,0 +1,44 @@
+    name: Analyse Code With CodeQL
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
+    steps:
+       - name: Checkout repository
+         uses: actions/checkout@v4
+
+       - name: Initialize CodeQL
+         uses: github/codeql-action/init@v2
```

actionlint (1.7.4)

the runner of "github/codeql-action/init@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)